uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -582,72 +569,70 @@ private function deleteDates() {
    *
    * @param int $contactId
    * @param \DateTime $fromDate
-   * @param string $fromType
+   * @param string $fromDateType
    * @param null|\DateTime $toDate
-   * @param null|string $toType
+   * @param null|string $toDateType
    *
    * @return array
    *   An array of formatted results
    */
-  public static function calculateBalanceChange($contactId, DateTime $fromDate, $fromType, DateTime $toDate = null, $toType = null) {
```

Comment:
The to_date and to_date_type is now required on the BAO. I think it should apply on this method also?
If then, we should probably update the parameters to required on the `LeaveRequest.calculateBalanceChange` endpoint and also on this method.